settings/common.py

```diff
@@ -331,3 +331,5 @@
 EKS_CLUSTER_ROLE_ARN = os.environ.get("EKS_CLUSTER_ROLE_ARN")
 
 EKS_NODEGROUP_ROLE_ARN = os.environ.get("EKS_NODEGROUP_ROLE_ARN")
+
+ENVIRONMENT = os.environ.get("ENVIRONMENT", "production")
```

Comment:
Let's set it to `dev` by default and add it in the production and staging env files.